components/raftstore/src/store/peer.rs

```diff
@@ -4776,6 +4782,7 @@ where
         req: RaftCmdRequest,
         check_epoch: bool,
         read_index: Option<u64>,
+        is_read_index: bool,
```

Comment:
Can we determine whether this is a read index request according to the above `read_index` option?

Reply:
No, because read_local also sets read_index. See `Peer::read_local`.